src/web/app/pages-instructor/instructor-search-page/instructor-search-page.module.ts

```diff
@@ -27,6 +28,7 @@ const routes: Routes = [
     InstructorSearchComponentsModule,
     RouterModule.forChild(routes),
     LoadingSpinnerModule,
+    Pipes,
   ],
 })
-export class InstructorSearchPageModule { }
```

Comment:
As has been pointed out previously, the convention is to leave a space in between the brackets, therefore there's no need to remove this.